components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/hardware-wallet-connect/index.tsx

```diff
@@ -166,13 +169,19 @@ export const HardwareWalletConnect = ({ onSuccess, selectedAccountType }: Props)
       stopIndex: accounts.length + DerivationBatchSize,
       scheme: selectedDerivationScheme,
       coin: selectedAccountType.coin,
-      network: filecoinNetwork
+      network: filecoinNetwork,
+      onAuthorize: setShowAuthorizeDevice
```

Comment:
This onSubmit function is called when the user attempts to connect a hardware wallet.  I pass an onAuthorize function which eventually is passed to the constructor function of the SolanaLedgerKeyring class, so that it can be called once it is notified via postMessage that authorization has completed.

Reply:
`HardwareWalletConnectOpts.onAuthorize` needs a strict type definition.